bin/configure-cli-tests.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
```

Comment:
That file lives in <marathon_dir>/bin but really is not intended for the user to run.
Will discuss this with you directly.